lib/smart_city/test_data_generator.ex

```diff
@@ -132,6 +132,7 @@ defmodule SmartCity.TestDataGenerator do
         }
       ],
       schema: Payload.get_schema(:test),
+      transformations: [],
```

Comment:
Do we want the TDG to be able to add a random transformation is one isn't specified?

Reply:
Discussed in teams, will add this in the future